unittests/test_settings_dist.py

```diff
@@ -0,0 +1,22 @@
+from .dojo_test_case import DojoTestCase, get_unit_tests_path
+import os
+
+basedir = os.path.join(get_unit_tests_path(), '..')
+
+
+class TestSettingsDist(DojoTestCase):
```

Comment:
I understand the desire to simplify here - there's no need to specify the hashcode fields if they can be covered by `DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL` - but I am hesitant about adding a test that string-parses the settings file like this. I think replacing the values in `HASHCODE_FIELDS_PER_SCANNER` and `DEDUPLICATION_ALGORITHM_PER_PARSER` as you have done above is sufficient.

Reply:
I understand your thoughts here. I implemented this to help contributors getting feedback on how to use deduplication like this PR shows https://github.com/DefectDojo/django-DefectDojo/pull/9185 

Shall I still remove it, then I can do it.

Reply:
I think you can remove this test and just add a brief blurb in the ["how to write a parser" docs](https://github.com/DefectDojo/django-DefectDojo/blob/master/docs/content/en/contributing/how-to-write-a-parser.md#deduplication-algorithm) recommending the use of a pre-defined deduplication algorithm if applicable.